foodsaving/stores/models.py

```diff
@@ -1,15 +1,24 @@
+from enum import Enum
+
 from django.conf import settings
 from django.db import models
 
 from foodsaving.base.base_models import BaseModel, LocationModel
 
 
+class StoreStatus(Enum):
+    CREATED = 'created'
+    NEGOTIATING = 'negotiating'
+    ACTIVE = 'active'
+    DECLINED = 'declined'
+    ARCHIVED = 'archived'
```

Comment:
I tried the Enum style, it seems ok, the most annoying thing is having to put `.value` on the end to get the actual string out, if you forget that when you pass it into a `.filter()` call, then nothing will match.